constants/index.js

```diff
@@ -24,10 +24,6 @@ export const CREATE_CLAIM_REQUEST = 'CREATE_CLAIM_REQUEST';
 export const CREATE_CLAIM_FAILURE = 'CREATE_CLAIM_FAILURE';
 export const CREATE_CLAIM_SUCCESS = 'CREATE_CLAIM_SUCCESS';
 
-export const SHOW_CLAIM_REQUEST = 'SHOW_CLAIM_REQUEST';
```

Comment:
Why did you remove these lines?

Reply:
Since we no longer to use the `showClaims` action, so we remove these related constants.

Redux = Action + Reducer
Redux holds the state in the one place.
When action dispatched, the reducer will call and change the Redux state.

Since the claim data only used in the `ShowClaimPage` page, the claim data is not necessary to behold in the Redux state.

The user profile and session token will suitable for Redux state when user login to our website.

Reply:
Indeed, mention in Git commit messages as well.

Reply:
I mention these into the commit message. 👍